### DISCUSSION TITLE
Suggest to provide information on how to use the disk storage for disk image

### DISCUSSION THREAD

**Discussion by wyatt-wong** [tjboldt/Apple2-IO-RPi#82] [General]
I am familiar with Raspberry Pi but I am not familiar with the Apple 2 IO RPi card. I would suggest to provide information on how to use the storage space for adding and running disk images

**Comment by tjboldt** [comment]
Currently there is no hot-swap of drive images but you can change which drive images will be used on next boot of the RPi. If you recently built the RPi microSD from scratch, you can see optional step 24 in the setup instructions in #63 but if you have the previous setup that uses the cronjob, you can edit which two drive images are used by editing the cronjob with `crontab -e`. You can copy ProDOS compatible drive images to the RPi by any means normally used to copy to a Linux system. Having a hot swap of drive images was planned but never implemented. You can see this in the code on line 27 of [driver.go](https://github.com/tjboldt/Apple2-IO-RPi/blob/main/RaspberryPi/apple2driver/driver.go) file.
